Doc/whatsnew/3.11.rst

```diff
@@ -1128,34 +1128,63 @@ fcntl
 Optimizations
 =============
 
-* Compiler now optimizes simple C-style formatting with literal format
-  containing only format codes ``%s``, ``%r`` and ``%a`` and makes it as
-  fast as corresponding f-string expression.
-  (Contributed by Serhiy Storchaka in :issue:`28307`.)
+This section covers specific optimizations independent of the
+:ref:`whatsnew311-faster-cpython` project, which is covered in its own section.
 
-* "Zero-cost" exceptions are implemented. The cost of ``try`` statements is
-  almost eliminated when no exception is raised.
+* "Zero-cost" exceptions are implemented, eliminating the cost
```

Comment:
Wasn't this one part of Faster CPython?

Reply:
I wasn't sure about that either, especially since it was @markshannon who implemented it, but it at least wasn't listed there (whereas another exception-related change, for catching an exception when it is raised, is). I'm happy to move it if so.

@markshannon @Fidget-Spinner could you confirm? If it was, I can just remove it here and add it to the Misc section in #98429 , next to the other exception change and alongside the `re` change mentioned below. Thanks!

Reply:
Yes, it was part of the Faster CPython project.

Reply:
Thanks, I went ahead and moved it too